### PR TITLE
Various fixes to the documentation

### PR DESCRIPTION
**Pull request by achmiel**
This pull request fixes multiple minor errors found in the documentation (#1322).

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: docs/wiki/deployment/logging.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ By adding an outer join of `time` and using `time.minutes` as a counter this que
 
 Snapshot logs are an alternate form of query result logging. A snapshot is an 'exact point in time' set of results, no differentials. If you always want a list of mounts, not the added and removed mounts, use a snapshot. In the mounts case, where differential results are seldom emitted (assuming hosts do not often mount and unmount), a complete snapshot will log after every query execution. This *will* be a lot of data amortized across your fleet.
 
-To be extra-super-clear about the burden of data snapshots impose they are logged to a dedicated sink. The **filesystem** logger plugins writes snapshot results to **/var/log/osquery/osqueryd.snapshots.log**.
+To be extra-super-clear about the burden of data snapshots: impose they are logged to a dedicated sink. The **filesystem** logger plugins writes snapshot results to **/var/log/osquery/osqueryd.snapshots.log**.
```
Comment (theopolis) [inline docs/wiki/deployment/logging.md]: This is meant to read: Data snapshots may generate _a large amount_ of output. For log collection safety, output is written to a dedicated sink.

File: docs/wiki/deployment/performance-safety.md [hunk 2]
```diff
@@ -72,14 +72,14 @@ The build will run each of the support operating system platform/versions and in
...
 * Run a performance measurement using `./tools/profile.py`
-* Check performance against the latest release tag and commit to master.
-* Build docs and API spec on release tag or commit to master.
+* Check performance against the latest release tag and commit to master
+* Build docs and API spec on release tag or commit to master
 
 ## Virtual table blacklist
 
-Performance impacting virtual tables are most likely the result of missing features/tooling in osquery. Because of their dependencies on core optimizations there's no hard including the table generation code in master as long as the table is blacklisted when a non-developer builds the tool suite.
+Performance impacting virtual tables are most likely the result of missing features/tooling in osquery. Because of their dependencies on core optimizations, there is no hard including the table generation code in master as long as the table is blacklisted when a non-developer builds the tool suite.
```
Comment (theopolis) [inline docs/wiki/deployment/performance-safety.md]: ", these is no harm"

File: docs/wiki/deployment/remote.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ We include a very basic example python TLS/HTTPS server: [./tools/tests/test_htt
 
 The TLS clients built into osquery use the system-provided OpenSSL libraries. The clients use boost's ASIO header-libraries through the [cpp-netlib](http://cpp-netlib.org/) HTTPS library. OpenSSL is very outdated on OS X (deprecated since OS X 10.7), but still salvageable. 
 
-On Linux and FreeBSD the TLS client prefers the TLS 1.2 protocol, but includes TLS 1.1 and TLS 1.0-- and the following cipher suites:
+On Linux and FreeBSD the TLS client prefers the TLS 1.2 protocol, but includes TLS 1.1 and TLS 1.0 and the following cipher suites:
```
Comment (theopolis) [inline docs/wiki/deployment/remote.md]: Could be written:

On Linux and FreeBSD the TLS client prefers the TLS 1.2 protocol, but includes TLS 1.1/1.0 as well as the following cipher suites:

**Comment by theopolis** [comment]
You are my hero! As most of the corrected grammar errors are my own, please accept a very personal thank you! 

The notes I made to the PR are not your errors, but rather opportunities for a few additional fixes if you don't mind. :smile_cat:

**Comment by achmiel** [comment]
Thanks for the suggestions, I updated the code and squashed my commits into one.

**Comment by theopolis** [comment]
You're the best!

**Comment by marpaia** [comment]
Thanks, @achmiel!

**Pull request merged**